plutus-core/testlib/PlutusCore/Generators/Hedgehog/Builtin.hs

```diff
@@ -100,6 +103,10 @@ genConstant tr
     , Just HRefl <- eqTypeRep trList (typeRep @[]) =
         case genConstant trElem of
             SomeGen genElem -> SomeGen $ Gen.list (Range.linear 0 10) genElem
+    | trList' `App` trElem <- tr
```

Comment:
@effectfully Does this look right to you?

Reply:
Yeah, looks correct to me. Except the range is somewhat short, but maybe that's how it's supposed to be, don't know. In the bright future I'll kill all of it and create a proper `QuickCheck` generator for all this stuff.

Reply:
> Except the range is somewhat short,
I just copied that from the case for `[]` without thinking.  I'm not sure exactly which tests these generators get used in, so it's not clear to me whether we need bigger lists.